storage/immunitycache/interface.go

```diff
@@ -0,0 +1,13 @@
+package immunitycache
+
+// CacheItem defines the interface of a cache item
+type CacheItem interface {
```

Comment:
this interface should be in storage/interface.go - no need for this file.

Reply:
Moved.